db/migrate/20230407191113_add_post_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: { to_table: :users}
+  end
```

Comment:
- [ ] To match the project requirements, I think it would be better if you update above migration as you adding `Post` to `Comment` that means comments belong to Post so for that the `foreign_key` will be to in the `posts` table and not in `users` table.
```ruby
  def change
    add_reference :comments, :post, null: false, foreign_key: { to_table: :posts}
  end
 ...
```
or only
```ruby
  def change
    add_reference :comments, :post, null: false, foreign_key: true
 ...
```

- [ ] Once you have fixed the above change, kindly make sure you update your `db:schema.rb` file. And to do that just follow the bellow instructions:
  1 - Delete the `db:schema.rb` file
  2 - Run `rails db:migrate:reset` (This will create a new updated `db:schema.rb` file).
But Good job :100: :clap: :clap: